### DISCUSSION TITLE
Minecraft crashing

### DISCUSSION THREAD

**Discussion by Divadej** [gnembon/fabric-carpet#1204] [Q&A]
When I try to run minecraft with carpet, optifabric and optifine it crashes. But when I run minecraft with only carpet or only optifine it works just fine. I tried all the 1.18 versions of carpet. Could anyone please help me?

**Comment by altrisi** [comment]
This is an optifabric issue.

**Reply by altrisi** [reply]
This has already been fixed on Optifabric's side.
